spack/repo/packages/py-pyprecice/package.py

```diff
@@ -12,14 +12,17 @@ class PyPyprecice(PythonPackage):
     C++ library preCICE.
     """
 
-    homepage = "https://www.precice.org"
+    homepage = "https://precice.org"
     git = "https://github.com/precice/python-bindings.git"
     url = "https://github.com/precice/python-bindings/archive/v2.0.0.1.tar.gz"
```

Comment:
Why is this specifically `v2.0.0.1`? If it's working ok, but looks strange to me.

Reply:
@ajaust could you please make sure this is correct?

Reply:
It is correct. The `url` property is set to a downloadable archive which is often different from the homepage or git URL. This allows Spack to automatically substitute the version number to any other version number that is defined in the package. Using this substitution, Spack can download the corresponding tar ball. By default, Spack prefers to download archives over `git clone` as one can verify the checksum.

More information is in the Spack documentation on the [Version URLs](https://spack.readthedocs.io/en/latest/packaging_guide.html#version-urls).